cnames_active.js

```diff
@@ -1567,7 +1568,7 @@ var cnames_active = {
   "pomidorus": "tatyshev.github.io/pomidorus",
   "popcorn-api": "nirewen.github.io/popcorn-api",
   "popper": "fezvrasta.github.io/popper.js",
-  "portafolioacadeller": "acadeller.github.io/portafolioacadeller", 
+  "portafolioacadeller": "acadeller.github.io/portafolioacadeller",
```

Comment:
```suggestion
  "portafolioacadeller": "acadeller.github.io/portafolioacadeller", 
```
Please revert changes that aren't for your request :)

Reply:
sorry my formatter did his job 😆